apigenerator/resources/1 - Project/2 - Database/conn_resolvers/pgsql/PgSqlConnectionResolver.py

```
@@ -23,6 +24,15 @@ def get_pgsql_connection_session():
         # This block creates engine and session for the database #
             conn = get_pgsql_connection_schema_internet()
             engine = sa.create_engine(conn)
+
+            schema = get_global_variable('pgsql_schema')
```

Ruff (0.11.9)

`get_global_variable` may be undefined, or defined from star imports

(F405)